Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -1124,7 +1124,7 @@ def _get_git_diff(self) -> FilesToCollect:
             logger.info('bucket upload: getting last commit from index')
             previous_commit = get_last_commit_from_index(self.service_account, self.marketplace)
             if self.branch_name == 'master':
-                current_commit = 'origin/master'
+                current_commit = os.getenv("CI_COMMIT_SHA") or "origin/master"
```

Comment:
why use a default here? I think it's safe to just take `os.environ["CI_COMMIT_SHA"]`

Reply:
Why not? I think it's better to have a default here

Reply:
DMd you